content/outcomes/cloud-native-development/general__twelve-factor-apps.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Twelve-Factor Apps
+weight: 10
+layout: single
+team:
+ - VMware Tanzu Labs
+---
+
+Cloud native app development has centered around the twelve factors described initially by the team behind the Heroku platform. The website [https://12factor.net](https://12factor.net/) provides a great bootstrapped understanding of how a cloud native application deals with aspects like logging, application state, and integration with external systems.
```

Comment:
Remove "bootstrapped" as I'm not sure how it fits into this sentence.

Reply:
e8cf7596dd03e6d36bd918df693263a6f89065be